src/components/SocialLinks/SocialLinks.jsx

```diff
@@ -1,5 +1,22 @@
 import React, { Component } from 'react'
-import { ShareButtons, ShareCounts, generateShareIcon } from 'react-share'
+import {
+  FacebookShareButton,
+  GooglePlusShareButton,
+  LinkedinShareButton,
+  TwitterShareButton,
+  TelegramShareButton,
+  RedditShareButton,
+  FacebookIcon,
+  TwitterIcon,
+  TelegramIcon,
+  GooglePlusIcon,
+  LinkedinIcon,
+  RedditIcon,
+  FacebookShareCount,
+  GooglePlusShareCount,
+  LinkedinShareCount,
+  RedditShareCount
+} from 'react-share'
```

Comment:
`react-share` has a [new API](https://github.com/nygardk/react-share/blob/HEAD/migrate-v1-to-v2.md) in v2.